open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        def test_filter_with_happy_case(self):
</pre><p>You should not say test with happy case. You can only say test with a certain input, e.g. an invalid input. Remove the unncessary with from the test case name. Please apply this to all the others.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>...  <code>test_filter_with_a_certain_input</code> -&gt; <code>test_filter_happy_case</code></p>